app/pods/components/agenda/agenda-overview/agenda-overview-item/template.hbs

```diff
@@ -46,37 +46,42 @@
           {{/if}}
 
         </div>
-        <div class="vlc-u-flex-float-right">
 
-          {{!--    PUBLICATION FLOW CALL TO ACTION --}}
+        {{#if currentSessionService.isOvrb}}
```

Comment:
moet admin dit ook kunnen ?

Reply:
Enige bedenking, bevestigingsmodal want anders kan je die per ongeluk starten
Aangezien we nog geen implementatie hebben om publicaties te verwijderen ...

Reply:
Ik heb de admin id bij in die "checkOvrbRights" gestopt. raar dat die bij het aanmaken er niet is bij gestoken. Zit er nu bij in.

Reply:
"to-treat" route 
Terecht, daarom had ik ook alles in comment gezet. 
Bij deze ook gedelete.

Reply:
Confirmation is wel functioneel anders. Dat zou ik aan de klant overlaten of ze dat willen of niet.